qm.te

```diff
@@ -30,7 +30,19 @@ unconfined_domain(ipc_t)
 
 qm_domain_template(qm)
 
+#########################################
+# IPC between QM containers
+#########################################
+
+require {
+    type qm_file_t;
+    type qm_container_ipc_t;
+    class sock_file write;
+}
 
+#============= qm_container_ipc_t ==============
+allow qm_container_ipc_t qm_file_t:sock_file write;
+allow qm_container_ipc_t qm_t:unix_stream_socket connectto;
```

Comment:
I think we can replace this by the [stream_connect_pattern](https://github.com/fedora-selinux/selinux-policy/blob/rawhide/policy/support/ipc_patterns.spt#L4-L8) from the fedora/selinux-policy, adding also `search_dir_perms` on the ipc directory:
```suggestion
stream_connect_pattern(qm_container_ipc_t, ipc_var_run_t, qm_file_t, qm_t);
```

Reply:
macro seems much cleaner, I will give a test for double check. Thanks @engelmi